### PR TITLE
Adding a 3d touch button.

### PR DESCRIPTION
**Pull request by KratikJain10**
Hello! @Spyware007
Myself Kratik.
I want to add a 3d touch button ,kindly merge my pr if its ok.
#hactoberfest2022.
Thank You.

### CODE REVIEW

**Comment by Spyware007** [comment]
Align the button in the center using css.
You can take reference from other's code.

**Comment by KratikJain10** [comment]
@Spyware007 its already aligned in center only in my css file there is a margin of 50% from every side. Kindly confirm it.when i run it ,its in center only in my browser.

File: public/Buttons/KratikJain10/style.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  box-shadow: rgb(0, 0, 0) 4px 4px 0 0,#000 4px 4px 0 1px;
+  box-sizing: border-box;
+  color: rgb(8, 8, 8);
+  cursor: pointer;
+  display: inline-block;
+  font-family: ITCAvantGardeStd-Bk,Arial,sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+  margin: 50% 50% 50% 50%;
```
Comment (Spyware007) [inline public/Buttons/KratikJain10/style.css:16]: Remove this line.

File: public/Buttons/KratikJain10/style.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.button-50 {
```
Comment (Spyware007) [inline public/Buttons/KratikJain10/style.css:1]: Add
body{
 display:flex;
 justify-content:center;
 align-items:center;
}

**Review by KratikJain10 — COMMENTED** [review]
resolving

**Comment by KratikJain10** [comment]
@Spyware007 Kindly check the file ,,i resolved the issues as guided by you,,. Thank you for helping out. Now if its ok ,kindly merge it ,so I can complete hactoberfest2022.
Thank You.

**Comment by KratikJain10** [comment]
@Spyware007 Kindly check the file ,,i resolved the issues as guided by you,,. Thank you for helping out. Now if its ok ,kindly merge it ,so I can complete hactoberfest2022.
Thank You.

**Comment by KratikJain10** [comment]
@Spyware007 please merge it as soon as possible ,as its my last pr for hactoberfest2022 and it will be in review period of 7 days.
Thank You

**Comment by Spyware007** [comment]
@KratikJain10 Amazing work. 👀
Thank You for your contribution. 🤩
Leave a 🌟 if you like.

**Comment by Spyware007** [comment]
@KratikJain10 Amazing work. 👀
Thank You for your contribution. 🤩
Leave a 🌟 if you like.

**Comment by KratikJain10** [comment]
@Spyware007 Thanks 😃.
looking forward to more contributions..

**Pull request merged**